shingled_disassembly/conservative_disasm.ml

```diff
@@ -0,0 +1,53 @@
+open Core.Std
+open Or_error
+open Bap.Std
+open Bap_plugins.Std
+module Dis = Disasm_expert.Basic
+
+exception Create_mem of Error.t
+module Lifted_image=Lifted_image_piqi
+
+let lifted_of_bil instruction length offset =
+  { Lifted_image.Lifted_instruction.instruction=Bil_piqi.stmts_to_piqi instruction; length; offset }
+
+let jump global_mem base_addr max_addr lift_insn state mem insn (current_elem, bil_disasm) =
```

Comment:
when you have so many parameters, it is better to coalesce it into a record type.